.env.example

```diff
@@ -1,7 +1,14 @@
-APP_URL=http://in.localhost:8003/
-APP_KEY=<insert your generated key in here>
+APP_URL=http://in.localhost:8004
+CNAME=in.localhost
+PORT=8004
+
+APP_KEY=docker run --rm -it invoiceninja/invoiceninja php artisan key:generate --show
```

Comment:
Let's not change this, I would imagine someone would use this as the actual app key. Rather and add a comment the line before.
```
# Run `docker run --rm -it invoiceninja/invoiceninja php artisan key:generate --show` and copy the output here
```